lib/CloudinaryPlugin.js

```diff
@@ -318,16 +291,39 @@ class CloudinaryPlugin {
    * 
    * @see https://cloudinary.com/documentation/image_upload_api_reference#tags_method
    */
-  async replace_tag(request){
-    const tag = this._getArg(request, 'tag'), to = this._getArg(request, 'public_ids').split(",");
-    return await Cloudinary.uploader.replace_tag(tag, to)
+  async replace_tag(request) {
+    const tag = this._getArg(request, 'tag'), to = this._getArg(request, 'public_ids').split(',');
+    const res = await Cloudinary.uploader.replace_tag(tag, to);
+    this._handleError(to, res.public_ids);
+    return res;
   }
 
+  /** Extracts the args with the given name from the given request. 
+   * If the request does not contains an attribute with the given name, it will throw a bad request error. 
+   * 
+   * @param {Request} request 
+   * @param {String} name 
+   */
   _getArg(request, name) {
```

Comment:
I don't think this helper is needed. Since you can perform the action easily like this:
```js
const { arg1, arg2 } = request.input.args;